webrender/src/frame.rs

```diff
@@ -557,12 +559,12 @@ impl Frame {
                                               info.image_rendering);
             }
             SpecificDisplayItem::Text(ref text_info) => {
+                let instance = context.resource_cache.get_font_instance(text_info.font_key).unwrap();
```

Comment:
It might make sense to store the font instance key in the text primitive here. Then, we wouldn't need the resource cache at all at this part of the code (it could be fetched later), and the text run primitive structure could be made significantly smaller. We could do this as a follow up though...